utils/cautils/bootstrap.go

```diff
@@ -25,9 +25,11 @@ import (
 )
 
 type bootstrapAPIResponse struct {
-	CaURL       string `json:"url"`
-	Fingerprint string `json:"fingerprint"`
-	RedirectURL string `json:"redirect-url"`
+	CaURL             string `json:"url"`
+	Fingerprint       string `json:"fingerprint"`
+	RedirectURL       string `json:"redirect-url"`
+	Issuer            string `json:"issuer"`
+	MinPasswordLength int    `json:"min-password-length"`
```

Comment:
We probably want `omitempty` and use provisioner instead of issuer
```
Provisioner string `json:"provisioner,omitempty"`
```

RedirectURL should also have an omitempty

Reply:
This is the response, so it can be ok here.